Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN mkdir -p /srv/lightchain && \
 
 RUN rm -rf $GOPATH/src/github.com/lightstreams-network/lightchain
```

Comment:
[MAJOR] Is this a good way to do it? On line 14 in this file, we copy everything into the image (including the vendors), I don't see the point of doing few lines later `deps ensure` if we copied the vendors already (which takes several minutes) and then remove it all.

This also increases the docker image size significantly AFIK because it leaves a "layer of all files, including all the vendors" inside the image (history, even if u call "rm -rf" later). I am not 100% sure but I think it works like that.

I would suggest to build the binary in a separate container to make our production docker image as light as possible.

Reply:
I did believe the same I tried out both versions without a significant impact on the size of the image. Although I think it does not bring any value removing that, even I would say it add the handicap of focing the user to re-fetch the repo. Therefore I will remove that statement